### PR TITLE
Kaa 926 Make KAA_MAX_LOG_LEVEL overridable for tests

### PR DESCRIPTION
**Pull request by Dubland**
/cc @rasendubi @forGGe

### CODE REVIEW

File: client/client-multi/client-c/build.sh [hunk 1]
```diff
@@ -30,9 +30,13 @@ then
     help
 fi
 
+if [ -z ${MAX_LOG_LEVEL} ]
```
Comment (rasendubi) [inline client/client-multi/client-c/build.sh]: The following solution is incorrect: http://stackoverflow.com/a/13864829/2538771

File: client/client-multi/client-c/build.sh [hunk 2]
```diff
@@ -30,9 +30,13 @@ then
...
+then
+    MAX_LOG_LEVEL=6;
```
Comment (rasendubi) [inline client/client-multi/client-c/build.sh]: `;` is not necessary

**Comment by rasendubi** [comment]
LGTM

**Comment by forGGe** [comment]
Looks good

**Pull request merged**